Tribler/community/multichain/statistics/page_rank.py

```diff
@@ -5,9 +5,19 @@
 
 
 class IncrementalPageRank(object):
-
     """
     Class to compute page ranks according to a graph.
+
+    Incremental PageRank is an approximation for the actual PageRank score. It is computed using
+    a Monte Carlo method whereby a random walks are performed across the graph. All these random
+    walks are then stored in nested lists after which the number of occurrences of every node are
+    counted. More random walks give a more precise approximation of the actual PageRank score.
+    
+    We choose R to be the number of random walks performed starting from every node. The stopping
+    probability is c.
```

Comment:
Add _at each hop of the walk_ after probability? Also causes a better alignment :)